tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,147 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: marketplaces-push-disk-images
```

Comment:
This should align as well so you went with keeping the vm so this should be `marketplacesvm-push-disk-images`

Reply:
Changed